bot/models/user.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.orm import declarative_base
+import sqlalchemy as db
+from .base import Base
+
+class User(Base):
+    __tablename__ = "users"
+
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
have you looked at the telegram user's id type? do you think the simple Integer would be enough?